### PR TITLE
create capability command (only required fields)

### PR DESCRIPTION
**Pull request by HrisyToteva**
Issue [#2731](https://github.com/dfds/cloudplatform/issues/2731)

Capability cration request

Test command:  go run main.go capability create newCapability --description "capability desc" --invitees '["emcla@dfds.com"]' --metadata '{"dfds.cost.centre": "ti-ferry"}'

### CODE REVIEW

File: cmds/capability/capability.go [hunk 1]
```diff
@@ -50,3 +66,24 @@ var capabilityByIdCmd = &cobra.Command{
...
+			fmt.Println("Missing id")
+			os.Exit(1)
+		}
+		capabilityStruct := selfservice.CapabilityRequest{
+			Name:         args[0],
+			Description:  description,
+			Invitees:     invitees,
+			JsonMetadata: metadata,
+		}
+		fmt.Println(capabilityStruct)
```
Comment (andreasfrisch) [inline cmds/capability/capability.go:84]: Do we want this and the following?
They look like debugging -- and if so, maybe we should check for the 'verbose' flag?

File: selfservice/selfserviceClient.go [hunk 2]
```diff
@@ -45,6 +49,36 @@ func (sc *SelfServiceClient) queryServer(url string) []byte {
 	return responseBody
 }
 
+func (sc *SelfServiceClient) queryServerPost(url string, body []byte) []byte {
```
Comment (andreasfrisch) [inline selfservice/selfserviceClient.go:52]: I am not a big fan of the name `queryServerPost`.
We have another function called `queryServer` which performs GET.
Semantically then, this function should be called `sendToServer` or something along those lines.

We may have to rename both functions to achieve a good name -- and I am okay with that

**Review by andreasfrisch — APPROVED** [review]
Have we considered accepting files as input; For example in this, providing a JSON file might be simpler than setting all the variables

File: cmds/capability/capability.go (hunk 1 above)
Comment (HrisyToteva) [reply cmds/capability/capability.go:84]: The first one is reasonable, but without the second one we don't have an output

File: selfservice/selfserviceClient.go (hunk 2 above)
Comment (HrisyToteva) [reply selfservice/selfserviceClient.go:52]: maybe one general function would be enough?

File: cmds/capability/capability.go (hunk 1 above)
Comment (andreasfrisch) [reply cmds/capability/capability.go:84]: We can also simply keep both. Up to you.
We will change how we do output at a later stage anyway :)

File: selfservice/selfserviceClient.go (hunk 2 above)
Comment (andreasfrisch) [reply selfservice/selfserviceClient.go:52]: That could also work. Up to you :)

**Pull request merged**